specification/datadog/resource-manager/readme.go.md

```diff
@@ -21,10 +21,20 @@ azure-arm: true
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2022-06
   - tag: package-2021-03
   - tag: package-2020-02-preview
 ```
 
+### Tag: package-2022-06 and go
+
+These settings apply only when `--tag=package-2022-06 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-06' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2022-06-01/$(namespace)
+```
+
```

Comment:
We no longer support releasing more packages in go track 1 SDK any more since it is deprecating. Therefore could you please revert the changes in this file?

Reply:
@ArcturusZhang  does this mean for latest Api version we will not have GO SDK?